app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -348,7 +351,7 @@ def redirect_path_for_switch_language
       end
 
       def redirect_path_after_create_page
-        if @page.redirects_to_external?
+        if @page.redirects_to_external? || !@page.editable_by?(current_alchemy_user)
```

Comment:
@mamhoff we already have a spec for the first conditional, in another spec file though https://github.com/AlchemyCMS/alchemy_cms/blob/master/spec/controllers/alchemy/admin/pages_controller_spec.rb#L543-L600